content/index.html.haml

```diff
@@ -61,6 +61,13 @@ homepage: true
   :image => {:src => expand_link('images/gsoc/jenkins-gsoc-transparent.png'), :height => "300px"},
   :call_to_action => {:text => 'More info', :href => expand_link('projects/gsoc/')}}
 
+-# Case Studies
+- slides << {:href => 'http://JenkinsIsTheWay.io/',
+  :title => 'Jenkins Case Studies',
+  :intro => "Experiences of Jenkins users from around the world showcasing how users are building, deploying, and automating great software with Jenkins.",
+  :image => {:src => expand_link('images/post-images/jenkins-is-the-way/jenkins-is-the-way.png'), :height => "285px"},
+  :call_to_action => {:text => 'More info', :href => 'http://JenkinsIsTheWay.io/'}}
```

Comment:
There is no HTTPs in this domain. It will hammer SEO for the website. I sugest changing the link to the announcement blogpost for now